src/AzSK.ADO/Framework/Models/AzSKSettings.ps1

```diff
@@ -21,7 +21,7 @@ class AzSKSettings {
 	[string] $WebhookSource;
 	[string] $AutoUpdateCommand;
 	[string] $BatchScanModule;
-	[AutoUpdate] $AutoUpdateSwitch = [AutoUpdate]::On;
+	[AutoUpdate] $AutoUpdateSwitch = [AutoUpdate]::off;
```

Comment:
why is this changed required?

Reply:
for testing otherwise its prompt for user interaction